src/compiler/factoryPublic.ts

```diff
@@ -3403,7 +3403,7 @@ namespace ts {
             node.emitNode = {} as EmitNode;
         }
 
-        return node.emitNode;
+        return node && node.emitNode;

```

Comment:
We shouldn't need this. We initialize the value if it isn't set two lines before this.